### DISCUSSION TITLE
how to change screen recording resolution

### DISCUSSION THREAD

**Discussion by Deyvi-dev** [ant-media/Ant-Media-Server#5654] [FAQs]
I'm using Ant Media Server Community and I would like to know why the screen share feature is limited to 480p recording, and how can I increase the video quality when using the screen share feature or native camera?

**Comment by mekya** [comment]
Hi @Deyvi-dev, 
There is a solution for your question and I think someone will respond you for this soon. I'm not available to give the response for now. 
I mean I am just acknowledging it for you because it took 3 days since you ask the question

FYI

Cheers
Oguz

**Reply by Deyvi-dev** [reply]
Hi, thanks for your reply, I resolved the initial issue by changing the settings in advanced settings ("heightRtmpForwarding": 1080,)


Now my recordings are of higher quality, although they are very pixelated and blurry, even increasing the bitrate limit the image looks horrible, if anyone can help me I will be eternally grateful!

**Reply by yashtandon113** [reply]
@Deyvi-dev 
Yes, you discovered the correct settings to alter the resolution in community edition.

For screen sharing quality, I believe that several elements, such as bit rate and FPS, are important, as they contribute to stream stability. Otherwise, a low frame rate can result in choppy and low-quality video.

Also, ensure that the stream's sending resolution is not smaller than the output stream's resolution (1080) in your situation. Because if the lower resolution will be transcoded to higher resolution then it can also cause the quality issue.

**Reply by Deyvi-dev** [reply]
I've already made sure and tested the bitrate and framerate and still the video seems to have some pixelated image compression, I really don't know how to solve it

**Reply by yashtandon113** [reply]
@Deyvi-dev 
Could you please share a step-by-step reproduce scenario as well as a common web page using which we run the test by screen sharing?

**Reply by Deyvi-dev** [reply]
@yashtandon113  Thanks for asking! here is the example, I have already tested it with other resolutions and the result is the same

https://github.com/ant-media/Ant-Media-Server/assets/68565306/54849e98-347b-452e-bb0f-2015441ee789

**Reply by mekya** [reply]
Hi @yashtandon113,

Do you have any update?

Regards
Oguz

**Reply by yashtandon113** [reply]
@Deyvi-dev 
Please correct me if I misunderstand.

In your shared video, it seems you are using any other video source to publish the stream instead of screen sharing. If this is the case, it also depends on the FPS, resolution, etc. of the source that you are selecting. So could you please confirm the scenario?
